src/persist/api/Recoverable.cpp

```diff
@@ -0,0 +1,46 @@
+#include "Recoverable.hpp"
+
+Recoverable::Recoverable(GlobalTestConfig* gtc){
+    // init Persistent allocator
+    // TODO: put this into EpochSys.
```

Comment:
Perhaps this TODO is outdated?

Reply:
No. I'll do this in a different PR. In short, multiple `Recoverable` can share one `EpochSys` instance.

Reply:
Oh I see. I was thinking of merging API section in EpochSys into Recoverable.